src/Chapter2.hs

```diff
@@ -336,7 +348,12 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b x
+  | a < 0 || b < 0 = []
+  | b < a = []
+  | a == b = take 1 (drop a x)
+  | a < b = take (b - a + 1) (drop a x)
```

Comment:
This 2 clauses look similar 🕵🏼 
In case of `a == b` the expression `b - a + 1` would be equal to `1` which is exactly like it is done in the previous clause, so you can unite two of this into just one 🙂